src/Microsoft.DotNet.Build.Tasks/Targets/publishtest.targets

```diff
@@ -36,8 +31,6 @@
   </ItemGroup>
 
   <Target Name="CopyTestToTestDirectory"
-    Inputs="@(PackagesConfig);$(TestRuntimePackageConfig);@(ReferenceCopyLocalPaths);@(Content);@(IntermediateAssembly);@(_DebugSymbolsIntermediatePath)"
```

Comment:
For the Outputs here I was kind of thinking about a similar Semaphore file. However given that you are going to still use the CallTarget on this the inputs/outputs aren't really necessary.